.gitignore

```diff
@@ -1,2 +1,3 @@
 .deps
 bin
+/.deps/
```

Comment:
I think this is now a duplicate here in the .gitignore - I added `.deps` already when adding the `pony-jason` dependency.